.env.example

```
@@ -18,3 +18,19 @@ USDC_ISSUER_MAINNET=GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN
 # Slippage tolerance for USDC→XLM DEX swap (basis points; 50 = 0.5%)
 USDC_XLM_SLIPPAGE_BPS=50
 # Minting contract: initialize with the SAC contract address for this issuer+code so mint_from_usdc matches the DEX asset; ensure a testnet path from that asset to XLM exists.
+
+# ── S3 / KYC document storage (B-062) ────────────────────────────────────────
+# S3 bucket for KYC document uploads (must be private — no public access)
+AWS_S3_KYC_BUCKET=acbu-kyc-documents
+# AWS region for S3 (defaults to AWS_REGION if set)
+AWS_S3_REGION=us-east-1
+# Explicit credentials — omit to use IAM role / instance profile in production
+AWS_ACCESS_KEY_ID=your-access-key-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_ACCESS_KEY_ID key should go before the AWS_S3_KYC_BUCKET key

(UnorderedKey)